File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        curr_majority: int = nums[0]
        curr_count: int = 1
        for i in range(1, len(nums)):
            if nums[i] == curr_majority:
                curr_count += 1
            elif curr_count == 1:
                curr_majority = nums[i]
            else:
                curr_count -= 1
        return curr_majority

File: Data Structures & Algorithms/remove-element/submission-1.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        uint8_t l_idx = 0;
        for(uint8_t i = 0; i < nums.size(); i++) {
            if(nums[i] != val) {
                nums[l_idx++] = nums[i];
            }
        }
        return l_idx;
    }
};